src/useradd.c

```diff
@@ -1997,7 +1997,8 @@ static void faillog_reset (uid_t uid)
 	int fd;
 	off_t offset_uid = (off_t) (sizeof fl) * uid;
 
-	if (access (FAILLOG_FILE, F_OK) != 0) {
+	struct stat st;
```

Comment:
Can you move this line just below the other definitions and leave a blank line with the if statement?